<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWindow</class>
 <widget class="QWidget" name="AuthWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlaxaFlow - Acceso</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ESTILO GENERAL */
QWidget {
    background-color: #FAFAFA;
    font-family: 'Segoe UI', sans-serif;
    font-size: 14px;
    color: #3E2723;
}
QLineEdit {
    background-color: #FFFFFF;
    border: 1px solid #D7CCC8;
    border-radius: 4px;
    padding: 10px;
    margin-bottom: 10px;
}
QLineEdit:focus {
    border: 2px solid #FFB74D;
}
QPushButton {
    border-radius: 4px;
    padding: 10px;
    font-weight: bold;
}
/* BOTONES PRINCIPALES (ACCION) */
QPushButton.btn-primary {
    background-color: #4E342E;
    color: white;
}
QPushButton.btn-primary:hover {
    background-color: #5D4037;
}
/* BOTONES SECUNDARIOS (LINKS) */
QPushButton.btn-link {
    background-color: transparent;
    color: #4E342E;
    text-decoration: underline;
}
QPushButton.btn-link:hover {
    color: #FFB74D;
}
/* LOGO */
QLabel#Logo, QLabel#Logo_2 {
    background-color: #4E342E;
    color: white;
    font-size: 24px;
    font-weight: bold;
    border-radius: 30px; 
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     
     <widget class="QWidget" name="page_login">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Logo">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>AF</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Iniciar Sesión</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="styleSheet">
             <string>font-size: 20px; font-weight: bold; margin-bottom: 20px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_login_email">
         <property name="placeholderText">
          <string>Correo Electrónico</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_login_pass">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Contraseña</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_login_action">
         <property name="text">
          <string>ENTRAR</string>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="class" stdset="0">
          <string>btn-primary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_go_to_register">
         <property name="text">
          <string>¿No tienes cuenta? Regístrate</string>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="class" stdset="0">
          <string>btn-link</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="page_register">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Logo_2">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>AF</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Crear Cuenta</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="styleSheet">
             <string>font-size: 20px; font-weight: bold; margin-bottom: 20px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_reg_nombre">
         <property name="placeholderText">
          <string>Nombre Completo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_reg_email">
         <property name="placeholderText">
          <string>Correo Electrónico</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_reg_pass">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Contraseña</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_register_action">
         <property name="text">
          <string>REGISTRARSE</string>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="class" stdset="0">
          <string>btn-primary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_go_to_login">
         <property name="text">
          <string>Volver a Inicio de Sesión</string>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="class" stdset="0">
          <string>btn-link</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>